content/doc/book/managing/cli.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:hide-uri-scheme:
+
+= Jenkins CLI
+
+
+////
+Pages to mark as deprecated by this document:
+
+https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+CLI
+https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+SSH
+////
+
+[NOTE]
+====
+This is still very much a work in progress
+====
+
+Jenkins has a built-in command line interface that allows you to access Jenkins
+from a script or from your shell. This is convenient for automation of routine
+tasks, bulk updates, trouble diagnosis, and so on.
+
+This interface is accessed via the Jenkins CLI client, which is a Java JAR file distributed with Jenkins.
+
+
+== Downloading the client
+
+You can download the JAR file for the client from the URL
+`/jnlpJars/jenkins-cli.jar` on your Jenkins server, for example
+`https://example.com/jnlpJars/jenkins-cli.jar`
+
+In theory, the CLI JAR is dependent on the version of Jenkins, but in practice,
```

Comment:
This document was still a work-in-progress and this stupid paragraph has been nuked.